spec/models/student_spec.rb

```diff
@@ -0,0 +1,326 @@
+# Context architecture
+#
+# TODO: Complete contexts
+#
+# - Tests on database structure and model
+# - CSV and YML upload
+#  - with no duplicates and no sections
+#  - with duplicates and no sections
+#  - with no duplicates and sections
+#  - with duplicates and sections
+#  - with no duplicates and one section
+#  - with duplicates and sections and update of a section
+#  - with an invalid file
+
+require 'spec_helper'
+require 'shoulda'
```

Comment:
We're trying to move away from using `shoulda`. Please write all new tests using the `expect` syntax.  Also, any additional `require`s for your tests should be in the `spec_helper` file, not in each individual spec file.  

For a general overview of rspec best practices please see:

http://www.betterspecs.org